src/services/navigationBar.ts

```diff
@@ -700,11 +703,11 @@ namespace ts.NavigationBar {
     }
 
     /** Flattens the NavNode tree to a list, keeping only the top-level items. */
-    function topLevelItems(root: NavigationBarNode): NavigationBarNode[] {
-        const topLevel: NavigationBarNode[] = [];
+    function primaryNavBarItems(root: NavigationBarNode): NavigationBarNode[] {

```

Comment:
Is the "primary" navbar the middle dropdown?

Reply:
Explained it the comment where the selection actually occurs. I was trying to be 'editor agnostic' but I will just update 'primary' to 'middle' since (I think) this is VS only anyway.

Reply:
What about something like "container"?